connectors/migrations/db/migration_15.sql

```diff
@@ -0,0 +1,9 @@
+-- Migration created on Sep 24, 2024
+CREATE TABLE IF NOT EXISTS "snowflake_configurations" (
+    "id"  SERIAL,
+    "createdAt" TIMESTAMP WITH TIME ZONE NOT NULL,
+    "updatedAt" TIMESTAMP WITH TIME ZONE NOT NULL,
+    "connectorId" INTEGER REFERENCES "connectors" ("id") ON DELETE SET NULL ON UPDATE CASCADE,
```

Comment:
should it be non-nullable and set to delete cascade instead ?

Reply:
Def right, I added a commit!
I put it in restrict since we decided to stop relying on cascade.